src/main/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/application/TelemetryConfig.kt

```diff
@@ -40,4 +40,9 @@ class TelemetryConfig {
             .map { obj: RequestTelemetry -> obj.properties }
             .orElse(emptyMap())
     }
+
+    @Bean
+    fun getOperationId(): () -> String? {
```

Comment:
This is nullable which is OK but do we want to log or do something if that happens ?

Reply:
Yeah good call, will update.